modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -372,10 +418,10 @@ private NetworkMessage prepareReplicaErrorResponse(HybridTimestamp requestTimest
     private void idleSafeTimeSync() {
         replicas.values().forEach(r -> {
             ReplicaSafeTimeSyncRequest req = REPLICA_MESSAGES_FACTORY.replicaSafeTimeSyncRequest()
-                    .groupId(r.groupId())
+                    .groupId(r.join().groupId())
```

Comment:
Let's skip uncompleted replicas there's no sense in sending safeTimeSyncRequests to them.

Reply:
fixed